srcalc/src/main/java/gov/va/med/srcalc/db/ResultsDao.java

```diff
@@ -38,12 +38,19 @@
     
     private final SessionFactory fSessionFactory;
     
+    /**
+     * Constructs an instance.
+     * @param sessionFactory the SessionFactory used to get the current session.
+     */
     @Inject
     public ResultsDao(final SessionFactory sessionFactory)
     {
         fSessionFactory = sessionFactory;
     }
     
+    /**
+     * @see org.hibernate.SessionFactory#getCurrentSession()
```

Comment:
If you actually run javadoc, this produces some near-useless output. The output could be improved by chaing this to `A convenience method to call {@link org.hibernate.SessionFactory#getCurrentSession()}.`, but maybe we should just make these methods private and not document them at all.